### PR TITLE
put codebase into dog namespace

### PR DESCRIPTION
**Pull request by swagween**
add `dog` namespace

### CODE REVIEW

File: DogTales/src/services/services.hpp
[the remark this answers is not in the document]
Comment (swagween) [reply DogTales/src/services/services.hpp:1]: I think I just missed it. Also, should `test_services`, `test`, and `build_version` be in namespace dog as well? The last is currently in `namespace dog_tales`

[the remark this answers is not in the document]
Comment (karnkaul) [reply DogTales/src/services/services.hpp:1]: Yup that's correct, all the code we add to this project should be in the same namespace (or a nested one).

[the remark this answers is not in the document]
Comment (karnkaul) [reply DogTales/src/services/services.hpp:1]: Remember to remove `dog::` qualifications after you add `Services` to the namespace. 😄

**Comment by karnkaul** [comment]
Taking over stale PR.

**Review by karnkaul — APPROVED** [review]

**Pull request merged**